helpers.js

```diff
@@ -1,19 +1,33 @@
 const util = require("util");
 const childProcess = require("child_process");
-const { access } = require("fs/promises");
-const fs = require("fs");
+const fs = require("fs/promises");
 
 const exec = util.promisify(childProcess.exec);
 
 async function assertPathExistence(path) {
   try {
-    await access(path, fs.constants.F_OK);
+    await fs.access(path, fs.constants.F_OK);
```

Comment:
Module `fs/promises` doesn't have the `constants` member, this probably throws an error `Uncaught TypeError: Cannot read properties of undefined (reading 'F_OK')`

Reply:
It seems to have constants and works... https://nodejs.org/dist/latest-v10.x/docs/api/fs.html#fs_fs_constants